test/python/test_pack.py

```diff
@@ -447,28 +457,119 @@ def test_using_both_flows(project_without_dependencies, pack_format, tmpdir):
     assert re.search(r'You use deprecated .+ files and .+ files at the same time', output)
 
 
-@pytest.mark.parametrize('pack_format', ['tgz', 'docker'])
-def test_build_in_docker_without_download_token_for_ee(project_without_dependencies, pack_format, tmpdir):
+@pytest.mark.parametrize('pack_format', ['tgz'])
+def test_build_in_docker_sdk_params_for_ee(project_without_dependencies, pack_format, tmpdir):
     if not tarantool_enterprise_is_used():
         pytest.skip()
 
     project = project_without_dependencies
 
+    # remove TARANTOOL_SDK_PATH from env
     env = os.environ.copy()
-    del env['TARANTOOL_DOWNLOAD_TOKEN']
+    del env['TARANTOOL_SDK_PATH']
 
-    cmd = [
-        os.path.join(basepath, "cartridge"),
-        "pack", pack_format,
-        "--use-docker",
-        project.path
+    # one and only one of this params should be specified
```

Comment:
Seems it should be a unit test, not integration (but it's only a recommendation)

Reply:
Yes, I've already written unit tests for this case.

Reply:
But it has been already implemented https://github.com/tarantool/cartridge-cli/pull/166/files#diff-7baecb9cbb2a69a0b0c1b57742e201aeR17

Reply:
I will remove this test from integration tests